app/src/main/AndroidManifest.xml

```diff
@@ -35,6 +33,24 @@
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+
+            <meta-data
+                android:name="com.robotemi.sdk.metadata.UI_MODE"
+                android:value="1" />
+            <meta-data
+                android:name="com.robotemi.sdk.metadata.UI_FLAG"
+                android:value="1" />
+        </activity>
+
+        <activity
+            android:name=".settings.SettingsActivity"
+            android:screenOrientation="landscape">
+            <meta-data
+                android:name="com.robotemi.sdk.metadata.UI_MODE"
+                android:value="1" />
+            <meta-data
+                android:name="com.robotemi.sdk.metadata.UI_FLAG"
```

Comment:
why is this meta data duplicated with line 41?

Reply:
These meta-data is for Activity only.